pkg/network/usm/sharedlibraries/watcher_test.go

```diff
@@ -218,63 +159,36 @@ func (s *SharedLibrarySuite) TestSameInodeRegression() {
 	require.NoError(t, os.Link(fooPath1, fooPath2))
 	fooPathID2, err := utils.NewPathIdentifier(fooPath2)
 	require.NoError(t, err)
+	require.True(t, fooPathID1 == fooPathID2)
```

Comment:
you should use `Equal` instead, as the error message will give more details
```suggestion
	require.Equal(t, fooPathID1, fooPathID2)
```

Reply:
Good catch, left this by mistake